stdparams.go

```diff
@@ -0,0 +1,38 @@
+// +build go1.7
+
+package httprouter
+
+import (
+	"context"
+	"net/http"
+)
+
+type paramsKey struct{}
```

Comment:
The benefit of `struct{}` is that it is guaranteed to be unique while an `int` could possible have a collision with another package